src/style.css

```diff
@@ -0,0 +1,9 @@
+.dropdown-menu {
+  --cui-dropdown-link-active-bg: #ebedef !important;
+  --cui-dropdown-link-active-color: rbga(44, 56, 74, 0.95) !important;
```

Comment:
We should probably document how package client can override style.

Reply:
shall i leave this for another PR? i would like to add styling and configuration last once the single select, multiselect and search things are all done, ticket: https://mrc-ide.myjetbrains.com/youtrack/agiles/103-61/current?issue=mrc-4697